data/tekton-pipelines/okd/windows11-customize-configmaps.yaml

```diff
@@ -1,59 +1,5 @@
 apiVersion: v1
```

Comment:
There is no only windows11-unattend CM left, is it of any use? Maybe we should drop this and the windows10 equivalent and only provide them as example in the docs?

Reply:
I would like to keep these two CMs and update them for our pipelines to skip all settings and get fully working vm without need to set anything

Reply:
What about keeping this CM in `windows-customize.yaml` then too? Or creating a new file for both CMs?

Reply:
makes sense to keep it only in single file

Reply:
I did a little refactoring in files. Now each pipeline has its own file, and configmaps are divided into files with the name of pipeline to which they belong